cat-boot-report-pdf/src/main/java/cc/catalysts/boot/report/pdf/elements/ReportTable.java

```diff
@@ -29,7 +29,9 @@
     private boolean border = DEFAULT_BORDER;
     private boolean noBottomBorder;
     private boolean noTopBorder;
-    private boolean placeLastBorder = true;
+    private boolean noInnerBorders = false;
+    private boolean placeFirstBorder = false;
+    private boolean placeLastBorder = false;
```

Comment:
and again: keep published stuff stable - this means don't change default values when adding new features

placeLastBorder was true and should stay that way

when i upgrade to the new version i expect that i get exactly the same result again without having to touch my code